sdk/search/test-resources.json

```diff
@@ -1,8 +1,111 @@
-    "parameters": {},
-    "variables": {},
-    "resources": [],
-    "outputs": {}
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "baseName": {
+          "type": "string",
+          "defaultValue": "[resourceGroup().name]",
+          "metadata": {
+              "description": "The base resource name."
+          }
+      },
+      "location": {
+          "type": "string",
+          "defaultValue": "[resourceGroup().location]",
+          "metadata": {
+              "description": "The location of the resource. By default, this is the same as the resource group."
+          }
+      },
+      "searchSku": {
+          "type": "string",
+          "defaultValue": "basic",
+          "allowedValues": [
+              "free",
+              "basic",
+              "standard"
+          ],
+          "metadata": {
+              "description": "The Search service SKU to create."
+          }
+      }
+  },
+  "variables": {
+      "searchServiceName": "[concat('azs-java-', parameters('baseName'))]",
+      "searchApiVersion": "2020-03-13",
+      "storageAccountName": "[concat('azsstg', parameters('baseName'))]",
+      "storageContainerName": "[concat('azscontainer', parameters('baseName'))]",
+      "storageApiVersion": "2019-06-01"
+  },
+  "resources": [
+      {
+          "name": "[variables('searchServiceName')]",
+          "type": "Microsoft.Search/searchServices",
+          "apiVersion": "[variables('searchApiVersion')]",
+          "location": "[parameters('location')]",
+          "sku": {
+              "name": "[parameters('searchSku')]"
+          },
+          "properties": {
+              "replicaCount": 1,
+              "partitionCount": 1,
+              "hostingMode": "default",
+              "publicNetworkAccess": "Enabled",
+              "networkRuleSet": {
+                  "ipRules": []
+              }
+          }
+      },
+      {
+          "name": "[variables('storageAccountName')]",
+          "type": "Microsoft.Storage/storageAccounts",
+          "apiVersion": "[variables('storageApiVersion')]",
+          "location": "[parameters('location')]",
+          "properties": {
+              "accessTier": "Hot",
+              "supportsHttpsTrafficOnly": true
+          },
+          "sku": {
+              "name": "Standard_LRS",
+              "tier": "Standard"
+          },
+          "kind": "StorageV2",
+          "resources": [
+            {
+              "type": "blobServices/containers",
+              "name": "[concat('default/', variables('storageContainerName'))]",
+              "apiVersion": "[variables('storageApiVersion')]",
+              "properties": {},
+              "dependsOn": [
+                "[variables('storageAccountName')]"
+              ]
+            }
+          ]
+      }
+  ],
+  "outputs": {
+      "AZURE_SEARCH_SERVICE_ENDPOINT": {
```

Comment:
There is an ongoing effort to change these to NOT preface with "AZURE_", and just "SEARCH_...". If that is too big a change, though, probably don't worry about it here.

Reply:
Easy change, will make it in the next commit.